aspnet/fundamentals/configuration.rst

```diff
@@ -179,17 +178,17 @@ Next, create the custom configuration provider by inheriting from ``Configuratio
 
 .. literalinclude:: configuration/sample/src/CustomConfigurationProvider/EntityFrameworkConfigurationProvider.cs
   :language: c#
-  :emphasize-lines: 9,18-30,36-37
+  :emphasize-lines: 9,18-30,37-38
 
 Note the values that are being stored in the database ("value_from_ef_1" and "value_from_ef_2"); these are displayed in the sample below to demonstrate the configuration is reading values from the database properly.
 
-By convention you can also add an ``AddEntityFrameworkConfigurationSource`` extension method for adding the configuration provider:
+By convention you can also add an ``AddEntityFrameworkConfiguration`` extension method for adding the configuration provider:
 
 .. literalinclude:: configuration/sample/src/CustomConfigurationProvider/EntityFrameworkExtensions.cs
   :language: c#
   :emphasize-lines: 9
 
-You can see an example of how to use this custom ``ConfigurationSource`` in your application in the following example. Create a new ``ConfigurationBuilder`` to set up your configuration providers. To add the ``EntityFrameworkConfigurationSource`` you first need to specify the data provider and connection string. How should you configure the connection string? Using configuration of course! Add an *appsettings.json* file as a configuration source to bootstrap setting up the ``EntityFrameworkConfigurationSource``. By adding the database settings to an existing configuration with other sources specified, any settings specified in the database will override settings specified in *appsettings.json*:
+You can see an example of how to use this custom configuration provider in your application in the following example. Create a new ``ConfigurationBuilder`` to set up your configuration sources. To add the ``EntityFrameworkConfigurationProvider``, you first need to specify the data provider and connection string. How should you configure the connection string? Using configuration of course! Add an *appsettings.json* file as a configuration source to bootstrap setting up the ``EntityFrameworkConfigurationProvider``. By adding the database settings to an existing configuration with other sources specified, any settings specified in the database will override settings specified in *appsettings.json*:
```

Comment:
> To add the ``EntityFrameworkConfigurationProvider``, you first need to specify the data provider and connection string

Nit: I would suggest: To configure the ``EntityFrameworkConfigurationProvider``, you first need to specify the EF data provider and connection string